api/ryml.i

```diff
@@ -390,6 +442,9 @@ public:
     void set_key_ref   (size_t node, c4::csubstr ref   );
     void set_val_ref   (size_t node, c4::csubstr ref   );
 
+    void _set_key(size_t node, c4::csubstr key, int more_flags=0);
+    void _set_val(size_t node, c4::csubstr val, int more_flags=0);
```

Comment:
Why was this added? These are generally not to be called from client code, so I'm curious what the remaining API does not cover, but this covers instead.

Reply:
Late key name binding basically.  `_set_key` is how `NodeRef` does this, and `NodeRef` is not currently exposed.  I haven't ended up needing `_set_val` at all.